phoenix-core/src/it/java/org/apache/phoenix/end2end/BaseQueryIT.java

```diff
@@ -90,7 +90,8 @@ public BaseQueryIT(String indexDDL) {
     @Before
     public void initTable() throws Exception {
          ts = nextTimestamp();
-        initATableValues(tenantId, getDefaultSplits(tenantId), date=new Date(System.currentTimeMillis()), ts);
+
+        initATableValues("aTable", tenantId, getDefaultSplits(tenantId), date=new Date(System.currentTimeMillis()), ts, getUrl());
```

Comment:
First param is tableName. For this class we want tableName to be "aTable" hence passing it directly.